{"09cd1fce678fbe30fe0a320d069036b7":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"abcfe39cec7446d8","reportUrl":"../1287//#testresult/abcfe39cec7446d8","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='10.1.68.202', port=49152): Max retries exceeded with url: /metrics (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8993195660>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1712626221941,"stop":1712626367109,"duration":145168}}]},"a46dd5fec5170ae1b5112e2bb246aae9":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":1},"items":[{"uid":"b0a15b66a12ac9c5","reportUrl":"../1287//#testresult/b0a15b66a12ac9c5","status":"passed","time":{"start":1712626222629,"stop":1712626442198,"duration":219569}}]},"cee6faeba82de98eaf891ec9dd9520e1":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":1},"items":[{"uid":"ab4e6a3bf0f14de3","reportUrl":"../1287//#testresult/ab4e6a3bf0f14de3","status":"passed","time":{"start":1712626212568,"stop":1712626302621,"duration":90053}}]}}